src/Raven.Server/Rachis/RachisConsensus.cs

```diff
@@ -953,6 +956,7 @@ public void AcceptNewConnection(Stream stream, Action disconnect, EndPoint remot
                         }
                     }
                     _clusterId = initialMessage.TopologyId;
+                    SetClusterBase(_clusterId);
```

Comment:
There are a bunch of more places where this is used, for example, when the value is changed on the fly using admin js console

Reply:
Still need to add this in more places